ranker/shared/ranker_obj.py

```diff
@@ -190,6 +190,14 @@ def get_rgraph(self, answer):
         """Get "ranker" subgraph."""
         rnodes = set()
         redges = []
+        dummy_ind = 0
+        #adds edges and nodes from support graphs into the answer edge bindings and node bindings
+        for i_analysis in range(len(answer.get('analyses',[]))):
+            for sg in answer['analyses'][i_analysis]['support_graphs']:
```

Comment:
You should `.get('support_graphs', [])` because they aren't guaranteed to exist.